### PR TITLE
[fix] Handle router logits in Qwen 3 moe and Qwen 3 omni moe for aux loss

### PR DESCRIPTION
**Pull request by ngquangtrung57**
## Commit Message Convention

Please follow our standardized commit message format:

- `[feat]` - New features or functionality
- `[fix]` - Bug fixes
- `[docs]` - Documentation changes only
- `[style]` - Code style changes (formatting, missing semicolons, etc.)
- `[refactor]` - Code refactoring without changing functionality
- `[perf]` - Performance improvements
- `[test]` - Adding or updating tests
- `[chore]` - Maintenance tasks, dependency updates, etc.
- `[ci]` - CI/CD configuration changes

**Examples:**
- `[feat] add qwen omni iterable dataset support`
- `[fix] resolve bagel model configuration error`
- `[docs] update training guide with YAML examples`

See [CONTRIBUTING.md](../CONTRIBUTING.md) for more details.

## CI/CD Checks

Your PR will automatically run the following checks:

- **Linting**: Code formatting with `black` (line-length=120) and import sorting with `isort`
- Run `pre-commit run --all-files` locally to verify before pushing

## Checklist

- [x] Follow commit message convention (see above)
- [x] Run `pre-commit run --all-files` and ensure all checks pass
- [x] Format your code with `black` (line-length=120) and `isort`
- [x] Add unit tests for new functionality
- [x] Update documentation as needed, including docstrings or example tutorials
- [x] Ensure all CI/CD checks pass

### CODE REVIEW

File: src/lmms_engine/models/qwen3_omni_moe/qwen3_omni_moe_ops.py [hunk 1]
```diff
@@ -108,6 +109,11 @@ def text_model_forward(
     output_hidden_states = (
         output_hidden_states if output_hidden_states is not None else self.config.output_hidden_states
     )
+    output_router_logits = (
+        output_router_logits
+        if output_router_logits is not None
+        else getattr(self.config, "output_router_logits", False)
+    )
```
[the remark this answers is not in the document]
Comment (ngquangtrung57) [reply src/lmms_engine/models/qwen3_omni_moe/qwen3_omni_moe_ops.py]: Sure, will fix this

**Review by kcz358 — APPROVED** [review]
LGTM, thanks
